apps/iOS-WKWebView-sample/WkWebView DemoUITests/WkWebView_DemoUITests.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  WkWebView_DemoUITests.swift
+//  WkWebView DemoUITests
+//
+//
+
+import XCTest
+
+class WkWebView_DemoUITests: XCTestCase {
+
+    override func setUpWithError() throws {
+        continueAfterFailure = false
+    }
+
+
+    func joinMeetingTest() throws {
+        let app = XCUIApplication()
+        app.launch()
+        app.buttons["Go to WebView"].tap()
+        let webView = app.webViews
+        _ = webView.textFields.element(boundBy: 0).waitForExistence(timeout: 10)
+        let meetingTitleField = webView.textFields.element(boundBy: 0);
+        meetingTitleField.tap()
+        let meetingTitle = UUID().uuidString
+        meetingTitleField.typeText(meetingTitle)
+        let nameField = webView.textFields.element(boundBy: 1)
+        nameField.tap()
+        let attendeeName = UUID().uuidString
+        nameField.typeText(attendeeName)
+        app.toolbars.buttons["Done"].tap()
```

Comment:
This might be stretch goal, but I think making each page as [code](https://github.com/xamarin-automation-service/xcuitest-pop-example/blob/master/ToDoListUITests/ToDoListUITests.swift#L11) might be clean

Reply:
Since there's only one test case for this app - and there is no plan to add more pages to this ios app, I think its actually simpler to just have this one test.